.github/workflows/cmake.yml

```diff
@@ -210,6 +211,14 @@ jobs:
             ldflags: -static
             codecov: ubuntu_gcc_s390x
 
+          - name: Ubuntu GCC S390X DFLTCC
+            os: ubuntu-latest
+            compiler: s390x-linux-gnu-gcc
+            cmake-args: -DCMAKE_TOOLCHAIN_FILE=cmake/toolchain-s390x.cmake -DWITH_DFLTCC_DEFLATE=ON -DWITH_DFLTCC_INFLATE=ON
+            packages: qemu gcc-s390x-linux-gnu libc-dev-s390x-cross
+            ldflags: -static
+            codecov: ubuntu_gcc_s390x
+
```

Comment:
I feel like S390X w/DFLTCC should also be tested with  `-DZLIB_COMPAT=ON`, and possibly also with ` -DWITH_SANITIZERS=ON`, what do you think?

Reply:
Sure, let's give it a try. I normally develop it with ASAN on real hardware, let's see how this will look like in QEMU.